configure.ac

```diff
@@ -115,6 +116,18 @@ fi
 AC_SUBST(LIBNOTIFY_CFLAGS)
 AC_SUBST(LIBNOTIFY_LIBS)
 
+dnl ---------------------------------------------------------------------------
+dnl - Check for libatspi
+dnl ---------------------------------------------------------------------------
+
+PKG_CHECK_MODULES(LIBATSPI, atspi-2 >= $LIBATSPI_REQUIRED_VERSION,
+          [AC_DEFINE(HAVE_LIBATSPI, 1, [Define if libatspi is available])
+          have_libatspi=yes], have_libatspi=no)
+AM_CONDITIONAL([HAVE_LIBATSPI], [test "x$have_libatspi" = xyes])
+PKG_CHECK_EXISTS([atspi-2 > 2.36.0],
+                 [AC_DEFINE([DESTROYING_ATSPI_LISTENER_DOES_NOT_CRASH], [1], [Define if libatspi does not have bug 22])],
+                 [])
+
```

Comment:
The user must be able to enable or disable this feature, then check if the library is available and show if this is enabled in the configuration summary.

Reply:
OK, will do

Reply:
done